toolchain/sem_ir/inst_namer.cpp

```diff
@@ -616,6 +526,116 @@ auto InstNamer::NamingContext::AddWitnessTableName(InstId witness_table_inst_id,
   AddInstNameId(name_id, suffix);
 }
 
```

Comment:
It's a bit surprising to have methods of `NamingContext` and `InstNamer` interleaved like this. Can these be moved up next to `QueueBlockInsts`?